implementations/blaze/main.cc

```diff
@@ -10,6 +10,25 @@
 #include <iostream>
 #include <vector>
 
+#define WARMUP_ITERATIONS 100L
+#define MAX_WARMUP_TIME 10000000000
+
+bool validate_all(auto context, auto instances, auto schema_template) {
```

Comment:
@michaelmior Shouldn't all of these be references? Otherwise you are copying them, and doing so while measuring?

Reply:
Probably. That's likely why Blaze got so much slower here.